doc/content/verification_validation_examples/cad_workflow/cad_workflow.md

```diff
@@ -0,0 +1,148 @@
+  style=display:block;margin-left:auto;margin-right:auto;width:130%;
+
+!col-end!
+
+!col! small=12 medium=4 large=3
+
+!media figures/cad_workflow_d1.png
+  id=DAGMC_mesh
+  caption=Model DAGMC mesh
+  style=display:block;margin-left:auto;margin-right:auto;width:130%;
+
+!col-end!
+!row-end!
+
+## OpenMC
+
+The OpenMC input files is as follows:
+
+!listing /test/tests/neutronics/cad_workflow/model.py language=python
+
+## Cardinal
+
+The Cardinal input files is shown below. The ([MoabSkinner](https://cardinal.cels.anl.gov/source/userobjects/MoabSkinner.html)) was chosen in this model to update the DAGMC geometry from SALAMANDER mesh directly ensuring meshes matching which is further discussed in [!cite](Eltawila2024PBNC) as well as the use of cell tallies.
+
+!listing /test/tests/neutronics/cad_workflow/openmc.i
+
+## SALAMANDER (MOOSE) Heat Transfer
+
+SALAMANDER input file is as follows:
+
+!listing /test/tests/neutronics/cad_workflow/solid.i
+
+## Execution
+
+To generate OpenMC xml files, run:
+
+```
+python model.py
+```
+
+Then to run the coupled calculation:
+
+```
+mpiexec -np 2 salamander-opt -i solid.i --n-threads=2
+```
+
+This will run SALAMANDER with 2 MPI processes and 2 OpenMP threads per rank. To run the simulation faster, you can increase the parallel processes/threads, or simply decrease the number of particles used in OpenMC. When the simulation has completed, you will have created a number of different output files:
+
+- `solid_out.e`, an Exodus output with the solid mesh and solution from the MOOSE/SALAMANDER input file.
+- `solid_out_openmc0.e`, an Exodus output with the OpenMC solution and the data that was ultimately transferred in/out of OpenMC.
+
+## Results
+
+&nbsp;
+
+The results of this simulation on the finest mesh are shown in [temperatures], [tritium_production], and [results]. More information - including a mesh sensitivity study - is available in [!cite](Eltawila2024PBNC).
+
+&nbsp;
+
+!row! style=display:inline-flex;
+!col! small=12 medium=4 large=3
+
+!media figures/cad_workflow_Temps.png
+  id=temperatures
+  caption=Temperature distribution result from [!cite](Eltawila2024PBNC)
+  style=display:block;margin-left:auto;margin-right:auto;width:130%;
+
+!col-end!
+
+!col! small=12 medium=4 large=3
+
+!media figures/cad_workflow_tritium_production.png
+  id=tritium_production
+  caption=Tritium production rate density result from [!cite](Eltawila2024PBNC)
+  style=display:block;margin-left:auto;margin-right:auto;width:130%;
+
+!col-end!
+!row-end!
+
+&nbsp;
+
+!table id=results caption=Results summary
+| Parameter (Units) | Value |
+| :- | :- |
+| Armor Max. Temp. (K)         | $1062.4$                   |
+| First Wall Max. Temp. (K)    | $1057.6$                   |
+| Breeder Max. Temp. (K)       | $987.4$                    |
+| Heat Source (W)              | $2.44 \times 10^5 \pm 3 \times 10^3$   |
+| Tritium Production (atoms/s) | $4.70 \times 10^{13} \pm 8 \times 10^{11}$ |
```

Comment:
I'd prefer having 3 columns with the units on the right hand side, but this gets the job done.